public/locales/en/translation.json

```diff
@@ -55,5 +55,21 @@
     "header30": "Federal and military documents",
     "list30-item1": "<strong>If you are a former federal employee,</strong> have your <em>Notice to Federal Employees About Unemployment Insurance</em> (Standard Form 8) ready.",
     "list30-item2": "<strong>If you served in the military in the last 18 months,</strong> have your <em>Certificate of Release or Discharge from Active Duty</em> (DD 214) ready."
+  },
+  "tab2": {
+    "header10": "1. Register with Benefit Programs Online",
```

Comment:
Can we have dots in the name? header1.0 seems more clear than header10.

Reply:
yeah good idea... additionally, for searchability, i think i'll stop using namespaces even though it looks a bit nicer in the translations.json. so it'll be "tab2-header1.0" as the key name

Reply:
er, no dots in the name but i'm going to switch to 1b 1c etc instead of 10 11 etc